public/app/plugins/datasource/influxdb/components/editor/query/influxql/visual/PartListSection.tsx

```diff
@@ -70,7 +70,7 @@ const Part = ({ name, params, onChange }: PartProps): JSX.Element => {
   };
   return (
     <div className={partClass}>
-      <button className={cx('gf-form-label', noRightMarginPaddingClass)}>{name}</button>(
+      <button className={cx('gf-form-label', noRightMarginPaddingClass)}>[{name}]</button>(
```

Comment:
```suggestion
      <button className={cx('gf-form-label', noRightMarginPaddingClass)}>{name}</button>(
```

Reply:
We don't need those square brackets. 
It should look like this 
![image](https://github.com/grafana/grafana/assets/820946/ee2d2149-fdc2-4e0f-99bc-d9774789547e)

But with the square brackets it looks like this:
![image](https://github.com/grafana/grafana/assets/820946/20bbbd57-a109-4a05-9a6a-151a9810f48f)


So those square brackets must be gone. When we remove those we have some tests `VisualInfluxQLEditor.test.tsx` failed. They just need to be updated with the new results. I explained it here https://github.com/grafana/grafana/pull/77102#issuecomment-1788523680